api_products.md

```diff
@@ -214,8 +214,10 @@ Product attributes:
 |--------------------|----------|-----------|--------------------------------------------------------------------------------------------------------------------|
 | `code`             | `string` | ""        | Scannable code / barcode                                                                                           |
 | `template`         | `string` | "default" | Identifier of the [code template](#code-template) used                                                             |
-| `transmissionCode` | `string` | null      | In case the POS cannot handle the scannable code / barcode, it contains a POS friendly code                        |
+| `isPrimary`        | `bool`   | null      | If a product has a code marked as primary this code is always presented to a POS instead of the scanned one. A product is not allowed to have more than one primary code. Optional. |
+| `transmissionCode` | `string` | null      | In case the POS cannot handle the scannable code / barcode, it contains a POS friendly code. Should not be used in ombination with primary codes.                        |
 | `encodingUnit`     | `string` | null      | [Unit](#supported-units) which is used as encoding within the scanned code. Overwrites the property of the product |
+| `specifiedQuantity`| `number` | null      | If set the code identifies the specified Quantity of Products instead of a single one. Optional.                   |
```

Comment:
```suggestion
| `specifiedQuantity`| `number` | null      | If set the code identifies the specified quantity of products instead of a single one. Optional.                   |
```